app/lib/submission_builder/return_header1040.rb

```diff
@@ -193,19 +203,16 @@ def spouse_name_control(intake)
 
     private
 
-    def spouse_prior_year_agi(tax_return, intake)
-      return nil unless tax_return.filing_jointly?
+    def spouse_prior_year_agi(intake)
```

Comment:
because of the fact that this method will now always add content if they're married filing jointly, we needed the "fall through" to return 0, so I refactored the order of the if statements.